index.bs

```diff
@@ -10057,7 +10007,8 @@ in the node.
 				while connected inputs and/or script references exist. Such
 				nodes SHOULD return <code>false</code> from
 				{{process()}} which allows the presence or absence of
-				connected inputs to determine whether active processing occurs.
+				connected inputs to determine whether the {{AudioWorkletNode}}s
+				[=actively processing=] occurs.
```

Comment:
`actively processing occurs.` => `that is actively processing.`

Reply:
I rephrased this, what was above was a bit bogus.